scripts/install.sh

```diff
@@ -5,65 +5,30 @@ home_dir="gnu-koryavov"
 scipts_dir="scripts"
```

Comment:
Please consider updating this file according to the upstream `install.sh`

Reply:
Ok, fixed